### ISSUE TITLE
[ERROR] [AlgoContainer] Cant calculate, returning -1

### ISSUE THREAD

**Issue by joseph373334** [nicehash/NiceHashMiner#2967]
I have searched through the "errorbenchmark" troubleshooting page, and searched google + reddit and I am not finding reference to this error that is repeating showing-up in logs, when benchmarking is started. 
I've run gminer's mine_etc.bat and it has no issue.
This happens with many/most of the GPU benchmarks. I have disabled AV and Firewall.

Running on Windows 10 Pro, Ryzen 5900x, and 2 Nvidia cards (GTX & RTX)

[log.1.txt](https://github.com/user-attachments/files/17536095/log.1.txt)
[log.txt](https://github.com/user-attachments/files/17536096/log.txt)

**Comment by joseph373334** [comment]
Addition:
I have enabled and have been utilizing both gpu for multiple encodings. I neglected to mention that I am indeed running with the "Precise" type each time I start a benchmark, and I close the application fully before relaunching and executing a new benchmark pass. With the most recent logs (log_0) only having the RTX card's benchmarks selected, and not the other Nvidia(GTX) card's.
_

Drivers(Chipset, Audio, GPUs, LAN) were updated as of yesterday, BIOS included (non-beta). I have used DDU 15 days ago and installed Nvidia drivers then with NVClean, and then yesterday; agreed reboots are a give in. Device Manager is clear of errors; is clear of hidden/legacy devices. Windows was update yesterday and as of today. chkdsk, DISM, SFC have also been run without error.
_

Already running the application as Admin (adjusted in Compatibility mode for all users); UAC disabled.
_

A/V: On this particular machine it is running Avast (sorry this is the one that got this particular A/V in my rotation of machines). In Avast I have made excludes for the NHM with wildcard*. I even had run those log's benchmarks with Avast disabled. Block History & Quarantine is clear.
_

FW: I have used https://github.com/henrypp/simplewall which manipulates the WFP; I went as far as manually assigning each of the miners as an referenced app and created the rule to allow each of those full-access in & out. I also attempted benchmarks with Simplewall disabled and had it keep Defender/Windows FW disabled. Windows Notifications & Windows Security panel confirm no firewall protection is active. (To be clear: I ran benchmarks with FWs disabled and A/V disabled at the same time).
_

Hardware FW is pfSense and this particular client has full access out; it uses [1.1.1.1](http://1.1.1.1/) for DNS resolution.
_

Already have reinstalled NHM, and utilized the zip/portable method as a clean slate (installing all plugins) for the last 2 days(FW+Exceptions adjusted to point to this portable NHM path).
